hoodie-bench/src/main/java/com/uber/hoodie/integrationsuite/job/action/result/JavaRDDResult.java

```diff
@@ -0,0 +1,10 @@
+package com.uber.hoodie.integrationsuite.job.action.result;
+
+import org.apache.spark.api.java.JavaRDD;
+
+public class JavaRDDResult extends ActionResult<JavaRDD> {
```

Comment:
same with these methods? I am unable to clear understand how these classes are going to evolve? if these will be passed to a validate method, then it should be enough to simply templatize that?